S/StaticArrays/Compat.toml

```
@@ -1,10 +1,19 @@
 ["0-0.10"]
-julia = ["0.7", "1"]
+julia = ["0.7", "1-1.10"]
 
-["0.11-1.3"]
+["0.11"]
+julia = "1-1.10"
+
+["0.12"]
```

Based on https://github.com/JuliaRegistries/General/actions/runs/5533534662/jobs/10097202254?pr=86950#step:16:9:
```suggestion
["0.12-0"]
```

---

Thanks! I was unable to read that output.